src/mingw-contrib.mk

```diff
@@ -0,0 +1,18 @@
+# This file is part of MXE. See LICENSE.md for licensing information.
+
+PKG             := mingw-contrib
+$(PKG)_WEBSITE  := https://download.pahaze.net/ARM/mxe/mingw-contrib/
+$(PKG)_DESCR    := mingw-contrib - Fixes for missing or outdated libraries in LLVM-MinGW
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 1.2
+$(PKG)_CHECKSUM := 3eeb0e7b169c49359a7caed81cba8b3f89704fe713a49b87c5784a707a8a34d7
+$(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.gz
+$(PKG)_URL      := https://download.pahaze.net/ARM/mxe/mingw-contrib/src/$($(PKG)_FILE)
+$(PKG)_DEPS     := cc
```

Comment:
shouldn't we simply use GitHub?

Reply:
Yeah, that's my bad. Leftover from wmffix.mk